src/Maestro/SubscriptionActorService/DarcRemoteFactory.cs

```diff
@@ -39,10 +44,15 @@ public async Task<IRemote> CreateAsync(string repoUrl, long installationId)
                 settings.GitType = GitRepoType.GitHub;
                 settings.PersonalAccessToken = await GitHubTokenProvider.GetTokenForInstallation(installationId);
             }
-            else
+            else if (repoUrl.Contains("dev.azure.com"))
```

Comment:
>repoUrl.Contains("dev.azure.com") [](start = 21, length = 33)

In general, "string.contains" is a bad idea.  I could call my github repository "dev.azure.com", I think, or it could be in there for some other reason.

We know these are URI's, why not parse them properly and check the host instead of just string.contains?

Reply:
Yeah good idea.